src/unlimited.cpp

```diff
@@ -784,6 +784,62 @@ void GenerateBitcoins(bool fGenerate, int nThreads, const CChainParams &chainpar
         minerThreads->create_thread(boost::bind(&BitcoinMiner, boost::cref(chainparams)));
 }
 
+/** This function searches the mempool for transactions that are recently acceptable into the mempools of other
+nodes and forwards any found to those nodes.
+*/
+void ForwardAcceptableTransactions(const std::vector<CTxChange> &changeSet)
+{
+    // Start by grabbing node refs to minimize the time that the list is locked.
+    std::vector<CNodeRef> nodes;
+    nodes.reserve(vNodes.size());
+
+    {
+        LOCK(cs_vNodes);
+
+        for (CNode *pNode : vNodes)
+        {
+            if (pNode)
+                nodes.push_back(CNodeRef(pNode));
+        }
+    }
+
+    LOG(MEMPOOL | NET, "Check %d TX applicability to %d nodes\n", changeSet.size(), nodes.size());
+    for (auto &txc : changeSet)
+    {
+        LOG(MEMPOOL | NET, "%s: A:%d->%d, D:%d->%d, AS:%d->%d, DS:%d->%d\n", txc.tx->GetHash().ToString(),
+            txc.prior.countWithAncestors, txc.now.countWithAncestors, txc.prior.countWithDescendants,
+            txc.now.countWithDescendants, txc.prior.sizeWithAncestors, txc.now.sizeWithAncestors,
+            txc.prior.sizeWithDescendants, txc.now.sizeWithDescendants);
+    }
+    // Iterate through all changed transactions and all nodes to see if the tx crosses the nodes reject/accept boundary
+    for (auto &txc : changeSet)
+    {
```

Comment:
why not add the LOG statement above into this lower loop rather than go through the loop twice?

Reply:
I wanted these logs to be separated from the "TX YYY is now acceptable to node XXX" logs to make the log more readable.  Next release, we can remove this log entirely I think.  But it makes sense to keep it now while this feature is deployed as experimental.